components/clp-py-utils/clp_py_utils/clp_metadata_db_utils.py

```
@@ -1,7 +1,8 @@
 from __future__ import annotations
 
+from contextlib import closing
```

Flake8 (7.2.0)

[error] 'contextlib.closing' imported but unused

(F401)